### PR TITLE
Fix tests that were never running, add a new test

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #78836
* __->__ #78843

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

### CODE REVIEW

File: test/test_overrides.py [hunk 1]
```diff
@@ -1117,7 +1118,7 @@ def __torch_function__(self, *args, **kwargs):
                 # Function that handles torch_function in C++
                 torch.abs(a)
 
-@skipIfCrossRef
+@unittest.skipIf(TEST_WITH_CROSSREF, "not run with crossref")
```
Comment (samdow) [inline test/test_overrides.py:1121]: How is this different from [skipIfCrossRef](https://github.com/pytorch/pytorch/blob/d0baf5792a01cc5554d884b8579cc82239b87510/torch/testing/_internal/common_utils.py#L813)?

**Review by samdow — APPROVED** [review]
LGTM! Stuff like this is making me think though to Richard's point on an earlier PR that I want to unify the tests but that can come later

File: test/test_overrides.py (hunk 1 above)
Comment (ezyang) [reply test/test_overrides.py:1121]: Apparently skipIfCrossRef doesn't work when applied to an entire test class

**Pull request closed without merging**